pkg/apis/v1alpha1/serviceexport.go

```diff
@@ -88,3 +92,159 @@ type ServiceExportList struct {
+	ServiceExportConditionAccepted ServiceExportConditionType = "Accepted"
+
+	// ServiceExportReasonAccepted is used with the "Accepted" condition when the
+	// condition is True.
+	ServiceExportReasonAccepted ServiceExportConditionReason = "Accepted"
+
+	// ServiceExportReasonNoService is used with the "Accepted" condition when
+	// the associated Service does not exist.
+	ServiceExportReasonNoService ServiceExportConditionReason = "NoService"
+
+	// ServiceExportReasonInvalidServiceType is used with the "Accepted"
+	// condition when the associated Service has an invalid type
+	// (per the KEP at least the ExternalName type).
+	ServiceExportReasonInvalidServiceType ServiceExportConditionReason = "InvalidServiceType"
+)
+
+const (
+	// ServiceExportConditionExported is true when the service is exported to some
+	// control plane or data plane.
+	//
+	//
+	// Possible reasons for this condition to be true are:
+	//
+	// * "Exported"
+	//
+	// Possible reasons for this condition to be False are:
+	//
+	// * "Pending"
+	// * "Failed"
+	//
+	// Possible reasons for this condition to be Unknown are:
+	//
+	// * "Pending"
+	//
+	// Controllers may raise this condition with other reasons,
+	// but should prefer to use the reasons listed above to improve
+	// interoperability.
+	ServiceExportConditionExported ServiceExportConditionType = "Exported"
+
+	// ServiceExportReasonExported is used with the "Exported" condition
+	// when the condition is True.
+	ServiceExportReasonExported ServiceExportConditionReason = "Exported"
+
+	// ServiceExportReasonPending is used with the "Exported" condition
+	// when the service is in the process of being exported.
+	ServiceExportReasonPending ServiceExportConditionReason = "Pending"
+
+	// ServiceExportReasonFailed is used with the "Exported" condition
+	// when the service failed to be exported with the message providing
+	// the specific reason.
+	ServiceExportReasonFailed ServiceExportConditionReason = "Failed"
+)
+
+const (
+	// ServiceExportConditionConflicted indicates that some property of an
+	// exported service has conflicting values across the constituent
+	// ServiceExports. This condition must be at least raised on the
+	// conflicting ServiceExport and is recommended to be raised on all on
+	// all the constituent ServiceExports if feasible.
+	//
+	//
+	// Possible reasons for this condition to be true are:
+	//
+	// * "PortConflict"
+	// * "TypeConflict"
+	// * "SessionAffinityConflict"
+	// * "SessionAffinityConfigConflict"
+	// * "AnnotationsConflict"
+	// * "LabelsConflict"
+	//
+	// When multiple conflicts occurs the above reasons may be combined
+	// using commas.
+	//
+	// Possible reasons for this condition to be False are:
+	//
+	// * "NoConflicts"
+	//
+	// Controllers may raise this condition with other reasons,
+	// but should prefer to use the reasons listed above to improve
+	// interoperability.
+	ServiceExportConditionConflicted ServiceExportConditionType = "Conflicted"
```

Comment:
ditto, not obvious benefits to change to "conflicted" compared with "Conflict"

Reply:
For parity with Gateway API and the [SIG-Architecture API conventions for status conditions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties), which I believe did not exist yet when these early conditions were initially added, which explicitly recommend past-tense verbs.